apps/meteor/app/api/server/v1/users.ts

```diff
@@ -555,6 +559,130 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.listByStatus',
+	{
+		authRequired: true,
+		validateParams: isUsersListStatusProps,
+		permissionsRequired: ['view-d-room', 'view-outside-room'],
+	},
+	{
+		async get() {
+			const { offset, count } = await getPaginationItems(this.queryParams);
+			const { sort, fields } = await this.parseJsonQuery();
+			const { status, roles, searchTerm } = this.queryParams;
+
+			const projection = {
+				name: 1,
+				username: 1,
+				emails: 1,
+				roles: 1,
+				status: 1,
+				active: 1,
+				avatarETag: 1,
+				lastLogin: 1,
+				type: 1,
+				reason: 0,
+				...fields,
+			};
+
+			const actualSort = sort || { username: 1 };
+
+			if (sort?.status) {
+				actualSort.active = sort.status;
+			}
+
+			if (sort?.name) {
+				actualSort.nameInsensitive = sort.name;
+			}
+
+			let match;
```

Comment:
![image](https://github.com/RocketChat/Rocket.Chat/assets/2263066/636416de-7c26-4fc3-9da2-79d05861a839)
A `let` declaration without type implicitly working as `let match: any`. We don't like implicit `any`s.

Reply:
Indeed! Thanks Tasso!  I've added the proper types 🚀

Reply:
holy cow, this image covered my whole screen lol